src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,7 +10,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_REMARK = new Prefix("r/");
+    public static final Prefix PREFIX_PROJECT = new Prefix("pr/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_EMPLOYEE = new Prefix("e/");
```

Comment:
Does this not clash with the PREFIX_EMAIL, as they both use the the same Prefix "e/".

Reply:
Yes, but unlike the previous case where project and phone could exist in the add command, there hypothetically shouldn't be a command that needs both prefixes (edit command uses initial index as employee)